checks.d/network.py

```diff
@@ -176,6 +176,8 @@ def _check_linux(self, instance):
                 metrics = self._parse_linux_cx_state(lines[2:], self.TCP_STATES['netstat'], 5)
                 for metric, value in metrics.iteritems():
                     self.gauge(metric, value)
+            except Exception:
+                self.log.exception("Error collecting connection stats.")
```

Comment:
Opps, sorry, I meant to do `except Exception as e:` and then log the Exception message `e`. If you end up doing the custom Exception class below, it'll look different obviously